cli/internal/run/global_hash.go

```diff
@@ -100,30 +132,16 @@ func calculateGlobalHash(rootpath turbopath.AbsoluteSystemPath, rootPackageJSON
 
 	globalFileHashMap, err := hashing.GetHashableDeps(rootpath, globalDepsPaths)
 	if err != nil {
-		return struct {
-			globalFileHashMap    map[turbopath.AnchoredUnixPath]string
-			rootExternalDepsHash string
-			hashedSortedEnvPairs []string
-			globalCacheKey       string
-			pipeline             fs.PristinePipeline
-		}{}, fmt.Errorf("error hashing files: %w", err)
+		return nil, fmt.Errorf("error hashing files: %w", err)
 	}
 
-	globalHashable := struct {
-		globalFileHashMap    map[turbopath.AnchoredUnixPath]string
-		rootExternalDepsHash string
-		hashedSortedEnvPairs []string
-		globalCacheKey       string
-		pipeline             fs.PristinePipeline
-	}{
+	return &GlobalHashable{
```

Comment:
Any reason we're using a pointer here instead of the struct directly? Mostly just asking out of curiosity

Reply:
I was using a pointer so I could return nil instead of an empty struct in error scenarios, but it's not really required